VHDL rendering of assertions without an error message seems broken.

I'm running the latest dev build: **commit 9fc35a953e6ea43343a27a477805a8031dec5f3d (HEAD -> dev, origin/dev, origin/HEAD)**

I just switched back to VHDL rendering to run a sim in GHDL, and I get compile errors, which seem to me to be sensible:

`/mnt/project_scratch/FPGA/Vitis/2020.2/fpga_raytracer/src/spinal/rayshader/./simWorkspace/tmp/job_1/RSCSoCTop.vhd:35851:122:error: primary expression expected
      assert (not ((ctrl_stages_2_valid and ctrl_process_redoUpA) and (not ctrl_stages_2_preCtrl_FROM_A))) = '1' report () severity FAILURE;
                                                                                                                         ^
/mnt/project_scratch/FPGA/Vitis/2020.2/fpga_raytracer/src/spinal/rayshader/./simWorkspace/tmp/job_1/RSCSoCTop.vhd:35852:116:error: primary expression expected
      assert (not ((ctrl_stages_2_valid and ctrl_process_stallIt) and ctrl_stages_2_preCtrl_FROM_A)) = '1' report () severity FAILURE;`

I don't think it's legal to have the 'report' part of a VHDL assertion without a message-string:

`assert (not ((ctrl_stages_2_valid and ctrl_process_redoUpA) and (not ctrl_stages_2_preCtrl_FROM_A))) = '1' report () severity FAILURE;
      assert (not ((ctrl_stages_2_valid and ctrl_process_stallIt) and ctrl_stages_2_preCtrl_FROM_A)) = '1' report () severity FAILURE;`

I think the correct syntax would be to remove the `report ()` part entirely if there is no message.

--- a/core/src/main/scala/spinal/core/internals/ComponentEmitterVhdl.scala
+++ b/core/src/main/scala/spinal/core/internals/ComponentEmitterVhdl.scala
@@ -776,7 +776,11 @@ class ComponentEmitterVhdl(
                 case `ERROR`    => "ERROR"
                 case `FAILURE`  => "FAILURE"
               })
-              b ++= s"""${tab}assert $cond = '1' report ($message) $severity;\n"""
+              if (message.length > 0) {
+                b ++= s"""${tab}assert $cond = '1' report ($message) $severity;\n"""
+              } else {
+                 b ++= s"""${tab}assert $cond = '1' $severity;\n"""
+              }
            }
         }
 

--- a/core/src/main/scala/spinal/core/internals/ComponentEmitterVhdl.scala
+++ b/core/src/main/scala/spinal/core/internals/ComponentEmitterVhdl.scala
@@ -779,7 +779,7 @@ class ComponentEmitterVhdl(
               if (message.length > 0) {
                 b ++= s"""${tab}assert $cond = '1' report ($message) $severity;\n"""
               } else {
-                 b ++= s"""${tab}assert $cond = '1' $severity;\n"""
+                b ++= s"""${tab}assert $cond = '1' $severity;\n"""
               }
            }
         }

